### ISSUE TITLE
A11y_ SQL Azure Data Studio _ Notebooks Python Dependencies Wizard _Screenreader: Screenreader is not announcing the information of the data grid which is below Install required kernel Dependencies.

### ISSUE THREAD

**Issue by Yash14j** [microsoft/azuredatastudio#12960]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yTCS;#BM_SQLAzureDataStudio_Oct2020; #SQL Azure Data Studio;;#FTP;#NVDA;#A11yMAS;#A11ySev2;#DesktopApp;#Win10;#WCAG1.3.1;#Benchmark;#AINotInScope;

### **Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup 
Windows Version: Windows10
NVDA Version 2019.3.1

### **Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Press “Alt+Windows+N” keys to open new notebook
4. Open the Command Palette by pressing CTRL+SHIFT+P 
5. Type “Python” into the text box that appears. Select “Configure Python for Notebooks” with the help of down arrow key  from the menu. Python Dependencies Wizard will appear.
6. Navigate to edit button with the help of tab key, press enter and fill the details.
7. Navigate to next button with the help of tab key and press enter.
8. Step 2 Install Dependencies page will get open
9. Launch nvda and navigate to previous button with the help of tab key
10. Now with the help of left or right arrow key or insert+5, try to access information of the data grid which is below Install required kernel Dependencies
11. Check whether Nvda announces data grid information or not

### **Actual:**
Screenreader is not announcing the information of the data grid which is below Install required kernel Dependencies with the help of left or right arrow key or insert+5.
**This issue is only observed with NVDA**
**Same issue observed on page:**

- Import Extension - Preview Data
- Import Extension - Modify Columns
- Import Extension - Summary

- Summary 

### **Expected:**
Screenreader should announce the information of the data grid which is below Install required kernel Dependencies with the help of left or right arrow key or insert+5 along with table reference as column row number and header for each cell
### **User Impact:**
Screenreader user will not get proper and complete information of the data grid which is below “Install required kernel Dependencies”.
### **Attachment:**
[#12960_A11y_ SQL Azure Data Studio _ Notebooks Python Dependencies Wizard _Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/5391031/12960_A11y_.SQL.Azure.Data.Studio._.Notebooks.Python.Dependencies.Wizard._Screenreader.zip)

**Comment by alanrenmsft** [comment]
@chlafreniere are you or anyone from your team working on this issue? we are getting close to the due date 😄

**Comment by chlafreniere** [comment]
Thanks for the reminder, @alanrenmsft. @lucyzhang929 would you mind taking a look at this please?

**Comment by lucyzhang929** [comment]
Hi @Yash14j, 
After Step 9 (Launch nvda and navigate to previous button with the help of tab key), I am able to have the screen reader announce the table by
1. Press Insert+space
2. Press up arrow

Or, after selecting a kernel in the Kernel Dropdown, I am able to get to the table by
1. Press Insert + space
2. Press down arrow

Is this sufficient or are you saying that the table contents need to be announced right away when a new kernel is selected in the dropdown?

**Comment by Yash14j** [comment]
After selecting a kernel in the Kernel Dropdown, Nvda  announce the table by
Press Insert + space
Press down arrow
hence closing the bug. Please refer the attachment.
![12960_closed](https://user-images.githubusercontent.com/67999442/100863070-0527fc00-34ba-11eb-9943-18a8192e2372.png)
